spec/system/identity/user_matches_an_identity_spec.rb

```diff
@@ -0,0 +1,102 @@
+# frozen_string_literal: true
+require "rails_helper"
+
+RSpec.feature "Get an identity", type: :system do
+  include SummaryListHelpers
+
+  scenario "User matches an identity", vcr: { record: :once } do
```

Comment:
Why do we use `vcr: { record: :once }` vs. `vcr: true` in the spec above?

Reply:
Yeah this should be more consistent really. I briefly entertained the idea that making the record mode explicit was easier to work with when needing to re-record a cassette (just a matter of changing the mode rather than writing out the entire hash). I'll write a PR which tidies this up and adds a command line switch for the record mode instead.